RELEASE_NOTES.rst

```diff
@@ -1,3 +1,77 @@
+coala-bears 0.10.0
+==================
+
+This release, we had 97 commits contributed by 42 contributors
+
+We have updated the coala website to provide a quick overview of bears and
+their settings. To access this, visit http://coala.io/#!/languages. We also
+have sample bear output on the front page for some common languages.
+
+We have started separating the bear dependencies from this package. To keep
+current installation behavior in the future, please use the pip package index
```

Comment:
nooooo, this is american english, damn it, stop getting rid of "u"!!! :D behavioUr!!!